### PR TITLE
Add Akan tilde keyboard and update Akan QX keyboard

### PR DESCRIPTION
**Pull request by amire80**
* Add tilde to have similarity to other tilde input methods.
* Add tests.
* Add \Q and \X to QX replacement to allow typing Q and X.

### CODE REVIEW

File: rules/ak/ak-qx.js [hunk 1]
```diff
@@ -1,24 +1,26 @@
...
 	'use strict';
 
-	var defs = {
+	var akQx = {
 		id: 'ak-qx',
-		name: 'QX replacement',
+		name: 'Akan QX replacement',
 		description: 'Akan input method with Q and X replaced by Ɛ and Ɔ',
 		date: '2016-06-23',
 		URL: 'http://www.kasahorow.org/node/260',
```
Comment (kartikm) [inline rules/ak/ak-qx.js]: https please!

File: test/jquery.ime.test.fixtures.js [hunk 2]
```diff
@@ -9,6 +9,30 @@ var palochkaVariants = {
...
 	testFixtures = [ // eslint-disable-line no-unused-vars
+	{
+		description: 'Akan QX test',
+		inputmethod: 'ak-qx',
+		tests: [
+			{ input: 'Q', output: 'Ɛ', description: 'Akan QX Q' },
+			{ input: 'q', output: 'ɛ', description: 'Akan QX q' },
+			{ input: 'X', output: 'Ɔ', description: 'Akan QX X' },
+			{ input: 'x', output: 'ɔ', description: 'Akan QX x' },
+			{ input: '\\Q', output: 'Q', description: 'Akan QX \\Q' },
```
Comment (kartikm) [inline test/jquery.ime.test.fixtures.js]: Should this be \Q instead of \\Q and so on?

Comment (amire80) [reply test/jquery.ime.test.fixtures.js]: JavaScript literal strings require '\\' when one backslash is needed.

Comment (amire80) [reply test/jquery.ime.test.fixtures.js]: WAT. I'm pretty sure I wrote two. Trying again: '\\'.

Comment (amire80) [reply test/jquery.ime.test.fixtures.js]: Funny GitHub, changes two backslashes to one.

Comment (amire80) [reply test/jquery.ime.test.fixtures.js]: Anyway, I'm pretty sure that the code is correct.

**Pull request merged**